ofl/nabla/METADATA.pb

```diff
@@ -14,6 +14,7 @@ fonts {
 }
 subsets: "latin"
 subsets: "latin-ext"
+subsets: "math"
```

Comment:
I believe that you mentioned that we need Vietnamese to display the `Ẽ` and `ẽ`?

Reply:
added in https://github.com/google/fonts/pull/5095/commits/01494285c496089345f8ae1795634f7a68bb6cda